zjit/src/cruby.rs

```diff
@@ -271,7 +271,7 @@ pub type IseqPtr = *const rb_iseq_t;
 #[derive(Clone, Copy, PartialEq, Eq, Debug)]
 pub struct ShapeId(pub u32);
 
-pub const INVALID_SHAPE_ID: ShapeId = ShapeId(RB_INVALID_SHAPE_ID);
+pub const INVALID_SHAPE_ID: ShapeId = ShapeId(!0);
```

Comment:
I think we can afford the storage for a C const. Can we leave this as a symbolic value imported from C please?